style.css

```diff
@@ -193,11 +264,17 @@ a:hover {
     }
 
     .gallery-column {
-        flex-basis: 90%;
+        flex-basis: 100%;
     }
 
-    .hero_item--desktop-only {
-        display: none;
+    .gallery-column img {
+        padding-top: 2em;
```

Comment:
Remove this and instead add `margin` on it's parent meaning the `gallery-column_image` element

Reply:
Also, remove the `margin-bottom` and `margin-top` from the `<img>` element. They should be on the parent.